about.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="./myAbout.css">
+    <title>Conference for Collaborative Software Development</title>
+</head>
+<body>
+    <div class="top-outline">
+        <ul class="outline-items">
+            <li><i class="fa fa-github"></i></li>
+            <li><i class="fa fa-facebook"></i></li>
+            <li><a href="#">Discover</a></li>
+            <li><a href="#">Login</a></li>
+            <li><a href="#">SignUp</a></li>
+            <li></li>
+        </ul>
+    </div>
+    <header class="topbar">
+        <nav class="navbar">
+            <ul class="nav-items">
+                <li class="desktop-logo">
+                    <img class="logo-img" src="./images/bgimages/topLogo.png" alt="cc-log" >
+                    <span class="logo-text-home">CC 2022 <br> Collaborative Software Development</span>
+                </li>
+
+                <li class="li-deco"><a href="index.html">Home</a></li>
+                <li class="li-deco"><a href="#about.html">About</a></li>
+                <li class="li-deco"><a href="#speakers">Speakers</a></li>
+                <li class="li-deco"><a href="#partners">Partners</a></li> 
+            </ul>
+            <div class="hamburger" id="hamburger-menu"><a href="#">&#9776;</a></div>
+        </nav>
+    </header>
+    <main class="aboutUs">
+        <div class="about-header">
+            <h1 class="header-main-text">CREATIVE COMMONS COLLABORATIVE SOFTWARE DEVELOPMENT SUMMIT 2022</h1>
+            <h2 class="header-support-text">&quot;Hello! Sharing world&quot;</h2>
+        </div>
+        <div class="inner-text">
+            <p class="about-inner-text">The collaborative software Development Summit brings together the community of
+                experts, academics, and activists who comprise the Creative commons affiliate 
+                network in a different country every two years. This year, we hope to expand our
+                invitation list including organizations and individuals. So if you are an active or engaged software development
+                professional, we hope you will consider joining us this year to build a stronger commons together.                 
+            </p>
+            <p class="generalContact">Please contact us per Email for any further questions about CC Global Pair Programming 2022!</p>
+            <p class="emailContact">ccsummit2022@cckenya.org</p>
+        </div>
+    </main>
+    <section class="essentialContent">
+        <div class="summitLogo">
+            <div>
+                <h2 class="creative">Creative Commons Global Summit 2022 Logo </h2>
+                <hr class="line-partners">
```

Comment:
- [ ] Please make sure that your logo here is contained in a box 

Your design:- 
![image](https://user-images.githubusercontent.com/74744787/181387126-a09e1abc-e8e0-45db-aea9-ccdc6a4c9f64.png)
Behance's:- 
![image](https://user-images.githubusercontent.com/74744787/181387208-668a83bc-7543-4772-b2a4-dcdadd5ac6d8.png)

- notice how there is a line under the logo box please make sure to implement this also